tools/integration_tests/multiwriters_and_multimounts/multiwriters_and_multimounts_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Provide integration tests for multi-writers writing to multiple GCSFuse mounts.
+package multiwriters_and_multimounts_test
```

Comment:
should we keep the package as multimounts and then put another package as multiwrites inside it.
It helps us in adding more multimount scenarios in future.

Reply:
SG. I will create multimounts under multiwriters because in future we may add more multiwriters scenarios in future with single mount.